src/index.js

```diff
@@ -145,6 +151,7 @@
 
     (window['requirejs'] = window['require'] = req)['config'] = function (conf) {
         baseUrl = conf.baseUrl || baseUrl;
+        onNodeCreated = conf.onNodeCreated;
```

Comment:
You need to use `conf['onNodeCreated']` otherwise clojure compiler will not do the right thing